etc/testing/travis.sh

```diff
@@ -53,8 +56,9 @@ else
     popd
 fi
 
-make launch-loki
-
+if [[ "${BUCKET}" = "PPS3" ]]; then
+    make launch-loki
+fi
```

Comment:
I guess this isn't that important since we don't anticipate many test changes in 1.x, but the bucket a test is in isn't guaranteed to be stable, right?

Reply:
No, it's not. TBH my motivation was trying to make CI run faster, and I figured that despite the risk that TestLoki moves to another bucket due to adding/removing tests, this change was relatively low-risk because a) we'll know right away because CI starts failing, and b) it's easy to fix: you just change the bucket.